<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doktor Kayıt İşlemleri</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>DoktorID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Adı</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Soyadı</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ünvan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>_______</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtAdi"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtSoyadi"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cmbUnvan"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmbUzmanlik"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Uzmanlık</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QListWidget" name="doktorList"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btKaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btTemizle">
      <property name="text">
       <string>Temizle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btTemizle</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
